Lib/test/test_email/test__header_value_parser.py

```diff
@@ -2154,11 +2154,11 @@ def test_get_group_one_invalid(self):
 
     def test_get_group_missing_final_semicol(self):
         group = self._test_get_x(parser.get_group,
-            ('Monty Python:"Fred A. Bear" <dinsdale@example.com>,'
+            ('Monty Python:"Fred A. Bear" <dinsdale@example.com>, '
              'eric@where.test,John <jdoe@test>'),
```

Comment:
What about the missing space before `John` on this and subsequent lines? I realize it's not the issue you're solving, but since you're adding spaces after commas, do you need one here, too? Or maybe the point of this test is to not have spaces after commas? I haven't looked at it closely yet.

Reply:
These tests is the only thing about which I am unsure.

Reply:
I'm not 100% positive, but I think it's okay in this case.  I would wait for comment from @bitdancer though.

Reply:
I think you should leave all the email tests alone.  Having a mixture of spaces after commas and no spaces after commas is good thing in these tests, and although I don't remember for sure, most likely intentional.